tests/test_actor.py

```diff
@@ -90,17 +89,6 @@ def test_performs_cleanup_tasks_when_exiting() -> None:
     assert len(actor.independent_cleanup_tasks) == 0
 
 
-def test_assert_has_cleanup_tasks_is_deprecated() -> None:
-    actor = Actor.named("Tester")
-
-    with warnings.catch_warnings(record=True) as w:
-        actor.has_cleanup_tasks(FakeAction())
-
-    assert issubclass(w[-1].category, DeprecationWarning)
-    assert "has_ordered_cleanup_tasks" in str(w[-1])
-    assert "has_independent_cleanup_tasks" in str(w[-1])
-
-
```

Comment:
I'm not actually sure how our tests were passing after we removed this method from the `Actor` class, but they weren't passing anymore! Removed this test since the deprecated method has been removed.

Reply:
I can review the old test in the morning to see how it was passing.